Makefile

```diff
@@ -22,3 +39,47 @@ dispatch-test: set-git-ssh
 .PHONY: test
 test:
 	./test/run-tests.sh
+
+bin/.placeholder:
+	mkdir -p bin/
+	touch bin/.placeholder
+
+bin/kind_$(KIND_VERSION): bin/.placeholder
+	curl -Lso bin/kind_$(KIND_VERSION) https://github.com/kubernetes-sigs/kind/releases/download/v$(KIND_VERSION)/kind-$(OS)-$(KIND_MACHINE)
+	chmod +x bin/kind_$(KIND_VERSION)
+
+.PHONY: bin/kind
+bin/kind: bin/kind_$(KIND_VERSION)
+	@ln -sf ./kind_$(KIND_VERSION) bin/kind
+
+bin/kubectl-kuttl_$(KUTTL_VERSION): bin/.placeholder
+	curl -Lso bin/kubectl-kuttl_$(KUTTL_VERSION) https://github.com/kudobuilder/kuttl/releases/download/v$(KUTTL_VERSION)/kubectl-kuttl_$(KUTTL_VERSION)_$(OS)_$(MACHINE)
+	chmod +x bin/kubectl-kuttl_$(KUTTL_VERSION)
+
+.PHONY: bin/kubectl-kuttl
+bin/kubectl-kuttl: bin/kubectl-kuttl_$(KUTTL_VERSION)
+	@ln -sf ./kubectl-kuttl_$(KUTTL_VERSION) bin/kubectl-kuttl
+
+bin/go-junit-report:
+	GOBIN=${PWD}/bin/ go get -u github.com/jstemmer/go-junit-report
+
+.PHONY: install-bin
+install-bin: bin/kind bin/kubectl-kuttl bin/go-junit-report
+
+.PHONY: create-kind-cluster
+create-kind-cluster: install-bin $(KUBECONFIG)
+
+.PHONY: delete-kind-cluster
+delete-kind-cluster:
+	kind delete cluster
+	rm $(KUBECONFIG)
+
+$(KUBECONFIG):
+	kind create cluster --wait 10s --config=test/kind/kubernetes-$(KUBERNETES_VERSION).yaml
+	mkdir -p $(ARTIFACTS)
+	kubectl kuttl test --artifacts-dir=$(ARTIFACTS) test/kind/init 2>&1 |tee /dev/fd/2 | go-junit-report -set-exit-code > dist/addons_setup_report.xml
```

Comment:
This knows to use the kind cluster you made beforehand? is there anyway of specifying the path to the kubeconfig? if so, can you do it? it's going to come in handy when we finish our remote kind cluster service

Reply:
Yes. See the definition of KUBECONFIG above.